langs/vlang.yaml

```diff
@@ -0,0 +1,29 @@
+id: "v"
+aliases:
+  - "vlang"
+name: "V"
+
+install:
+  manual: |
+    git clone https://github.com/vlang/v
+    pushd v
+    make
+    popd
+
+main: "main.v"
+template: |
+  fn main() {
+    println('Hello, world!')
+  }
+
+compile: |
+  v run main.v
```

Comment:
If there's no separate compilation step, this can go under `run` instead of `compile`.